litestar/app.py

```diff
@@ -434,6 +436,7 @@ def __init__(
         self.debug = config.debug
         self.pdb_on_exception: bool = config.pdb_on_exception
         self.include_in_schema = include_in_schema
+        self.otel: DefineMiddleware | None = self._get_otel_middleware(config.middleware)
```

Comment:
I'm hesitant to introduce more special casing here for a specific middleware. We're generally trying to move away from this.

Reply:
can you please elaborate more on this ?

Reply:
I am asking this cause i was thinking about different way to extract the `otel` middleware from the list of middleware's with out the need to import `OpenTelemetryInstrumentationMiddleware` since we just want to get the type and nothing model cause a string comparison  will do the job.